crates/re_viewer_context/src/space_view/scene_element_list.rs

```diff
@@ -0,0 +1,84 @@
+//! Conversion from and to [`crate::Scene`] for tuples of [`crate::SceneElement`]s.
```

Comment:
…and only for tuples up to cardinality 2? What is the purpose of these conversions?

Reply:
convenience. I didn't need more so far and will need a tuple for them. But I can comment a bit more on the why :)